todo_app/data/trello_items.py

```diff
@@ -0,0 +1,91 @@
+import requests, os
+from todo_app.trello_config import Config
+
+def build_auth_query():
+    return {'key' : Config.TRELLO_API_KEY, 'token' : Config.TRELLO_API_TOKEN}
+
+BASE_URL= "https://api.trello.com/1/"
+
+
+def get_trello_board_id():
+    boards = requests.get(f"{BASE_URL}members/me/boards", params=build_auth_query())
+    boards_json = boards.json()
+    for list in boards_json:
+        if list['name'] == "Aaronboard":
+            return list["id"]
+    #return Config.TRELLO_BOARD_ID
+#print(get_trello_board_id())
+#
+def get_trello_todo_listid():
+    lists = requests.get(f"{BASE_URL}boards/{get_trello_board_id()}/lists", params=build_auth_query())
+    lists_json = lists.json()
+    for list in lists_json:
+        if list['name'] == "To Do":
+            return list['id']
+#print(get_trello_todo_listid())
+
+def get_trello_doing_listid():
+    lists = requests.get(f"{BASE_URL}boards/{get_trello_board_id()}/lists", params=build_auth_query())
+    lists_json = lists.json()
+    for list in lists_json:
+        if list['name'] == "Doing":
+            return list['id']
+#print(get_trello_doing_listid())
+
+def get_trello_done_listid():
+    lists = requests.get(f"{BASE_URL}boards/{get_trello_board_id()}/lists", params=build_auth_query())
+    lists_json = lists.json()
+    for list in lists_json:
+        if list['name'] == "Done":
+            return list['id']
+#print(get_trello_done_listid())
+
+class Item:
+    def __init__(self, id, title, status='To Do'):
+        self.id = id
+        self.status = status
+        self.title = title
+        #self.lastmodifieddate = lastmodifieddate
+
+
+def get_trello_cards():
+    cards = requests.get(f"{BASE_URL}boards/{get_trello_board_id()}/cards/", params=build_auth_query())
+    cards_json = cards.json()
+    items = []
+    for card in cards_json:
+        if card['idList'] == get_trello_todo_listid():
+            cardstatus = "To Do"
+        elif card ['idList'] == get_trello_doing_listid():
+            cardstatus = "Doing"
+        elif card ['idList'] == get_trello_done_listid():
+            cardstatus = "Done"
```

Comment:
Since `get_trello_todo_listid` etc. make API calls this loop makes up to 3 requests per card which can be slow (My test board has 20 cards on, and loading your app makes about 50 requests to Trello!). You don't need to change it to pass, but it's worth thinking about how you could make fewer requests and improve performance.